### PR TITLE
Update instructions for trebuchet

### PR DESCRIPTION
**Pull request by mosuem**
This will probably need more revisions over time.

---

- [x] I’ve reviewed the contributor guide and applied the relevant portions to this PR.

<details>
  <summary>Contribution guidelines:</summary><br>

- See our [contributor guide](https://github.com/dart-lang/.github/blob/main/CONTRIBUTING.md) for general expectations for PRs.
- Larger or significant changes should be discussed in an issue before creating a PR.
- Contributions to our repos should follow the [Dart style guide](https://dart.dev/guides/language/effective-dart) and use `dart format`.
- Most changes should add an entry to the changelog and may need to [rev the pubspec package version](https://github.com/dart-lang/sdk/blob/main/docs/External-Package-Maintenance.md#making-a-change).
- Changes to packages require [corresponding tests](https://github.com/dart-lang/.github/blob/main/CONTRIBUTING.md#Testing).

Note that many Dart repos have a weekly cadence for reviewing PRs - please allow for some latency before initial review feedback.
</details>

### CODE REVIEW

File: pkgs/trebuchet/bin/trebuchet.dart [hunk 1]
```diff
@@ -165,16 +165,26 @@ class Trebuchet {
...
-      'follow up with a PR adding links to the top-level readme table',
-      'transfer issues by running `dart run pkgs/repo_manage/bin/report.dart transfer-issues --source-repo dart-lang/$input --target-repo dart-lang/$target --add-label package:$input --apply-changes`',
-      'update the auto-publishing settings on pub.dev/packages/$input',
-      "add a commit to https://github.com/dart-lang/$input/ with it's readme pointing to the monorepo",
-      'archive https://github.com/dart-lang/$input/',
+      'Move and fix workflow files and badges in the README.md',
+      'Update pubspec.yaml to point to the new repository',
+      'Rev the version of the package, so that pub.dev points to the correct site',
+      'Add the package to the top-level readme of the monorepo',
+      "Merge the PR with 'Create a merge commit', enable `Allow merge commits` if not yet enabled on this repository",
```
Comment (devoncarew) [inline pkgs/trebuchet/bin/trebuchet.dart:174]: Perhaps,

```suggestion
      "**Important!** Merge the PR with 'Create a merge commit' (enabling then disabling the `Allow merge commits` admin setting)",
```

**Review by devoncarew — APPROVED** [review]

File: pkgs/repo_manage/lib/issue_transfer.dart [hunk 2]
```diff
@@ -42,13 +42,13 @@ class TransferIssuesCommand extends ReportCommand {
     argParser.addOption(
       'add-label',
       help: 'Add a label to all transferred issues.',
-      valueHelp: 'package:foo',
+      valueHelp: 'package-foo',
```
Comment (devoncarew) [inline pkgs/repo_manage/lib/issue_transfer.dart:45]: I believe we normally use colons in the package label.

Comment (mosuem) [reply pkgs/repo_manage/lib/issue_transfer.dart:45]: I saw that the `labeler.yml` adds `package-foo` labels, so wanted to adapt this. I am completely fine with either

Comment (devoncarew) [reply pkgs/repo_manage/lib/issue_transfer.dart]: Gotcha; we've used `package:foo` in our other monorepos (as far as I'm aware), so I'd like to do the same here.

**Comment by devoncarew** [comment]
Is this good to land? It would be good to have the updated instructions for package move PRs. (Note that the CI config in this repo has changed quite a bit since this PR was created.)

**Pull request merged**